src/components/design-system.css

```diff
@@ -263,16 +263,19 @@ a:hover {
   .fancyS {
     transition-delay: 0s;
   }
-  .get-an-estimate.show-when-scrolled {
+  .cta-button.show-when-scrolled {
     /* When not scrolled, is transparent and starts to the right */
-    transition: transform ease-in-out 0.7s, opacity ease-in-out 0.7s;
+    transition: transform ease-out 0.2s, opacity ease-out 0.2s;
```

Comment:
quicken the animation out to prevent overlap of cta button and logo on left unrolling from O to full logo